selfdrive/camerad/cameras/camera_qcom2.cc

```diff
@@ -64,7 +64,7 @@ const int ANALOG_GAIN_REC_IDX = 0x6; // 0.8x
 const int ANALOG_GAIN_MAX_IDX = 0xD; // 4.0x
 
 const int EXPOSURE_TIME_MIN = 2; // with HDR, fastest ss
-const int EXPOSURE_TIME_MAX = 1904; // with HDR, slowest ss
+const int EXPOSURE_TIME_MAX = 1618; // with HDR, slowest ss, 40ms
```

Comment:
Going over the 40ms of the readout doesn't seem to affect anything. 

Going over 50ms causes camera malfunctions and 10fps